examples/automm/distillation/README.md

```diff
@@ -0,0 +1,80 @@
+Examples showing how to use `AutoMMPredictor` for model distillation.
+
+## 1. Distillation on GLUE tasks
+
+### 1.1 Example
+[`automm_distillation.py`](./automm_distillation.py) : This example provide a use case 
```

Comment:
provide -> provides

Reply:
Have merged for now but @FANGAreNotGnu feel free to submit subsequent PRs to fix the typos! Thanks!

Reply:
I'll fix the typo in next PR, and also provide more experiment results. Thanks!